src/modules/account/components/AccountMenuListing/AccountMenuListing.js

```diff
@@ -8,35 +8,45 @@ import { Link } from 'react-router-dom';
 import { ApplicationBootstrapContext } from '@setup/react/app/ApplicationBootstrap';
 import { accountMenu } from '@account/const';
 import { useAuth } from '@auth/hooks/queries';
+import { useTokenBalances } from 'src/modules/token/fungible/hooks/queries';
 
 const AccountMenuListing = ({ className, onItemClicked }) => {
   const { t } = useTranslation();
   const [currentAccount] = useCurrentAccount();
   const { hasNetworkError, isLoadingNetwork } = useContext(ApplicationBootstrapContext);
+  const tokenBalances = useTokenBalances({
+    options: { enabled: !isLoadingNetwork && !hasNetworkError },
+  });
   const { address, isHW } = currentAccount?.metadata || {};
 
   const { data: authData } = useAuth({
     config: { params: { address } },
   });
 
-  function getDialogProps(component) {
+  function getDialogProps(component, data) {
     if (component === 'removeSelectedAccount' && currentAccount?.metadata?.address) {
       return {
         component,
-        data: { address: currentAccount?.metadata?.address },
+        data: { address: currentAccount?.metadata?.address, ...data },
       };
     }
-    return { component };
+    return { component, data };
   }
 
   return (
     <ul className={className}>
-      {accountMenu(authData, isHW, hasNetworkError, isLoadingNetwork).map(
-        ({ path, icon, label, component, isHidden }) =>
+      {accountMenu({
+        authData,
+        isHW,
+        hasNetworkError,
+        isLoadingNetwork,
+        hasAvailableTokenBalance: tokenBalances.data?.data?.length > 0,
```

Comment:
```suggestion
        hasAvailableTokenBalance: tokenBalances.data?.data?.length > 0,
```
This should be a check against available balance > 0? similar to other place where we have `.some( balance greater than 0`?

Reply:
e.g tokenBalances.data?.data?.some(
    ({ availableBalance }) => BigInt(availableBalance) > 0
  );